src/log.cpp

```diff
@@ -459,13 +459,8 @@ void log_printf(logLevel level, const char* format, ...) {
     log_file.close();
 
     /* Write the log message to the shell */
-    std::cout << msg_string;
+    printf(msg_string.c_str());
   }
-
-
-  /* If the message was a runtime error, exit the program */
-  if (level == ERROR)
-    exit(1);
 }
```

Comment:
I'll watch my StackOverflow (post)[http://stackoverflow.com/questions/29371292/exit-swig-wrapped-c-c-routine-without-killing-ipython-kernel] over the next few days to see if any SWIG wizards out there can help with this.